<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseUpdaterDialog</class>
 <widget class="QDialog" name="DatabaseUpdaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Object DB</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../3rd_party/QSimpleUpdater/etc/resources/qsimpleupdater.qrc">
    <normaloff>:/icons/update.png</normaloff>:/icons/update.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="browse_button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTextBrowser" name="errors"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Path to file:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="requirements">
     <property name="text">
      <string>Requirements to file content</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../3rd_party/QSimpleUpdater/etc/resources/qsimpleupdater.qrc"/>
 </resources>
 <connections/>
</ui>
